src/Model/ModelManager.php

```diff
@@ -297,6 +298,8 @@ public function getUrlSafeIdentifier($document)
 
     public function addIdentifiersToQuery($class, ProxyQueryInterface $queryProxy, array $idx)
     {
+        \assert($queryProxy instanceof ProxyQuery);
```

Comment:
Why do you need to add this assert?

Reply:
$queryProxy is a `ProxyQueryInterface` which does not have `getQueryBuilder`.

Reply:
But after thinking about it, it's maybe better to throw an exception when it's not a ProxyQuery instead of just using `assert`.

Reply:
I've removed adding new assertions, I'll add them and exceptions when raising phpstan level.